### DISCUSSION TITLE
Standards harmonization internationally

### DISCUSSION THREAD

**Discussion by shawna-slh** [w3c/wai-mission#16] [Global focus]
Ideally organizations around the world work together to develop and adopt the same accessibility standards.

When that doesn't happen and different standards are developed:
- Existing tools, technical guidance, supporting material, and training that are available internationally don’t cover different standards
- Multi-national organizations spend more time learning and following different standards, or, don’t meet local standards
- Local companies’ products that meet local standards might not meet international standards

How can we better encourage and support international uptake of WCAG in appropriate situations?
* [WCAG 2.0 is an ISO standard](https://www.w3.org/WAI/standards-guidelines/wcag/faq/#iso) and W3C tentatively plans to take WCAG 2.2 through the ISO process ([W3C PAS FAQ](https://www.w3.org/2010/04/pasfaq)).
* what else?

How do we address situations where it would be good to provide extensions to WCAG to support specific situations, such as different parameters for making non-Latin scripts more accessible to people with low vision and cognitive disabilities (e.g., [WCAG 1.4.12 Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html))?

How do we better engage with organizations around the world to develop future standards that work well internationally?

**Comment by mitchellevan** [comment]
I see good value of this proposed effort, but I have not upvoted it because I would rather see resources invested elsewhere.

Arguably WAI has enough reputation and relevance that it's likely that regional standards bodies would at least refer to WAI offerings in developing their own. They also have the option of sending representatives to W3C's groups and other range of participation options. Finally, some regions have the option of funding W3C for a project (e.g., as Europe has done in the past).

**Comment by maryjom** [comment]
I have upvoted this because I have seen some of the issues caused by non-harmonized standards where officially translated versions were not available and countries developed their own non-harmonized standards (e.g. Korea and China). I have also spoken to standards organizations from China, and the publication of WCAG as an ISO standard increased its standing in their eyes and they began work to bring their standard closer into alignment.

**Comment by mitchellevan** [comment]
Thanks @maryjom , I was not aware of the Korea and China precedents. I have up-voted after all.
